src/sections/Landscape/LandscapeGrid.style.js

```diff
@@ -11,7 +11,7 @@ export const LandscapePageWrapper = styled.div`
 		color:${props => props.theme.text};
 	}
 	#service-mesh-comparison-strength {
-            margin: -2.5rem auto 6rem;
+            margin: 0.5rem 1.2rem 5rem;
```

Comment:
Only update the `left, right` margin no need to update the `top, bottom` margin, i.e, `margin: -2.5rem 1rem 5rem` should be fine.

Reply:
Okay I'll update it